tests/test_common.js

```diff
@@ -512,3 +512,30 @@ test('testing timers are deleted correctly', t => {
     t.end()
   })
 })
+
+test('correct node behavior', t => {
```

Comment:
Could this test title capture the stated goal? 

This is from the original PR:

> The test makes sure http.request is not called when using http.get.

… on a mocked request? Is that it?

Nock has accumulated so much unusual code over the years to work around other libraries quirks, and it’s important that we can document why things are done the way they are. Same goes for a test. You’ll find a lot of tests like “filteringScope works” but it’s not as helpful in the future as saying the behavior that is wanted.

Reply:
Having dug into this a little more, I think this is not the right test. I think what we want to test is that the overridden / original `http.request()` is invoked only once by `http.get()`, instead of twice. That's the stated bug of #1836.

I'm going to dig in a bit and see if I can express that in a new test.

Reply:
I might have expressed myself a bit poorly in #1836. The main problem is that the native node behavior is not calling _req.request_ when _req.get_ is called. Nock does that in _common.js_. When using nock together with another library like newrelic that also overwrites these functions, you get a wrong call.

Reply:
This test checks that you don't call _req.request_ when _req.get_ is called, which is the native node behavior. There is not really anything else to test here in my opinion. I'll rename the test to what you suggested.

Reply:
Hey, thanks, would appreciate your help to clarify at https://github.com/nock/nock/issues/1836#issuecomment-582096922.